src/cfml/system/config/box.schema.json

```diff
@@ -378,8 +378,32 @@
 						}
 					},
 					"format": "uri"
+				},
+				"labels": {
+				    "title": "Labels List",
+					"description": "A list of labels to only include when running the tests",
+					"type": [
+						"string",
+						"array"
+					],
+					"items": {
+						"description": "A label a spec must have to be ran",
+						"type": "string"
+					}
+				}
```

Comment:
@elpete Missing comma here

Reply:
@KamasamaK Thanks for catching that.  I've just pushed that commit so it will be part of the next CommandBox 5.0.0-RC.1 build.